README.md

```diff
@@ -127,8 +125,18 @@ To build Ceph, follow this procedure:
 
        ninja install
 
-    
+## Build Types
+
+``do_cmake.sh`` by default creates a "debug build" of Ceph (assuming .git exists).
+A ``Debug`` build which can be up to five times slower than a non-debug build.
+Pass ``-DCMAKE_BUILD_TYPE=RelWithDebInfo`` to ``do_cmake.sh`` to create a
+non-debug build.
+The default build type is ``RelWithDebInfo`` once .git does not exist.
 
+| CMake mode          | Debug info | Optimizations   | Sanitizers          | Checks                   | Use for           |
+| ------------------- | ---------- | ----------------|-------------------- | -------------------------| ------------------|
+| `Debug`             | Yes        | `-Og`           | None                | `ceph_assert`, `assert`  | gdb. development  |
+| `RelWithDebInfo`    | Yes        | `-O2`, DNDEBUG  | None                | `ceph_assert`  only      | production        |
```

Comment:
If the intent is to note respective command line flags, then `DNDEBUG` should probably also have `-` and be highlighted the same way:
```suggestion
| `RelWithDebInfo`    | Yes        | `-O2`, `-DNDEBUG` | None                | `ceph_assert` only      | production        |
```